package.json

```diff
@@ -38,5 +38,9 @@
     "karma-firefox-launcher": "^1.1.0",
     "karma-mocha": "^1.3.0",
     "mocha": "^4.0.1"
+  },
+  "dependencies": {
+    "npm": "^5.8.0",
+    "update": "^0.7.4"
```

Comment:
Is this intentional? 
Did you forget `-g` when updating npm?

Reply:
Not intentional, I'm not paying attention :/
Any way to revert this?

Reply:
It's possible via `rebase -i` or cherry-pick + force-push

Reply:
Thx - rewriting historic, the head first